tests/result_set_tests.py

```diff
@@ -109,6 +109,13 @@ def test_int64_with_missing_data(self):
         results = SupersetResultSet(data, cursor_descr, BaseEngineSpec)
         self.assertEqual(results.columns[0]["type"], "BIGINT")
 
+    def test_data_from_list_of_lists(self):
+        data = [["a"], ["b"]]
+        cursor_descr = [("user_id", "STRING", None, None, None, None, True)]
```

Comment:
We can add another test for cursor descriptions that use `List[List[]]`

Reply:
Good point, I'll change the description to list of list here to tackle both cases in one test.